packages/vue/src/stories/getting-started/tailwind-css.stories.mdx

```diff
@@ -0,0 +1,295 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+import { YouTube } from 'mdx-embed';
+
+<Meta title="Setup/Integrations/TailwindCSS" />
+
+# TailwindCSS
+
+[TailwindCSS](https://tailwindcss.com/) is a utility-first CSS framework packed with classes like flex, pt-4, text-center and rotate-90 that can be composed to build any design, directly in your markup.
+You can use the it to customize **StorefrontUI** components in your application.
+
+<YouTube youTubeId="uCxPb_39TNg" />
+
+## Vue 2 based projects
+
+* Install TailwindCSS packages
+
+```bash
+yarn add -D tailwindcss@npm:@tailwindcss/postcss7-compat postcss@^7 autoprefixer@^9
+```
+
+* Generate `tailwind.config.js`
+
+```bash
+npx tailwindcss init
+```
+
+This will generate basic config file
+
+```js
+// tailwind.config.js
+
+module.exports = {
+  purge: [],
+  darkMode: false, // or 'media' or 'class'
+  theme: {
+    extend: {},
+  },
+  variants: {
+    extend: {},
+  },
+  plugins: [],
+}
+```
+
+* Create index.css file with TailwindCSS styles and include it in main.js
+
+```bash
+cd src && touch index.css
+```
+
+```css
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+```
+
+```js
+import Vue from 'vue'
+import App from './App.vue'
+import "./index.css";  // <-
+import "@storefront-ui/vue/styles.scss";
+
+Vue.config.productionTip = false
+
+new Vue({
+  render: h => h(App),
+}).$mount('#app')
+```
+
+
+## Nuxt.js based projects
+
+* Install TailwindCSS packages
+
+```bash
+yarn add -D @nuxtjs/tailwindcss autoprefixer postcss tailwindcss
+```
+
+* Add to nuxt build modules
+
+```js
+// nuxt.config.js
+
+export default {
+  buildModules: ['@nuxtjs/tailwindcss']
+}
+```
+
+* Generate `tailwind.config.js`
+
+```bash
+npx tailwindcss init
+```
+
+This will generate basic config file
+
+```js
+// tailwind.config.js
+
+module.exports = {
+  purge: [],
+  darkMode: false, // or 'media' or 'class'
+  theme: {
+    extend: {},
+  },
+  variants: {
+    extend: {},
+  },
+  plugins: [],
+}
+```
+
+* Add important rule to `tailwind.config.js` to override Storefront UI styles
+
+```js
+// tailwind.config.js
+
+module.exports = {
+  purge: [],
+  darkMode: false, // or 'media' or 'class'
+  theme: {
+    extend: {},
+  },
+  variants: {
+    extend: {},
+  },
+  plugins: [],
+  important: true,
+}
+```
+
```

Comment:
I would add something like this here: 
```<div class="alert warning">
   It is important to notice here, that adding `important` rule here will override also other classes 
written by yourself, so be aware that there are no conflicting classes that you use except of Tailwind.
</div>```

Reply:
What about adding here warning style? like this 
![Screenshot from 2021-09-08 11-12-18](https://user-images.githubusercontent.com/32042425/132482119-c63b86d7-7e8d-4326-9c91-f403118bea80.png)